src/StatusQ/Controls/Validators/StatusMinLengthValidator.qml

```diff
@@ -6,6 +6,12 @@ StatusValidator {
 
     name: "minLength"
 
+    errorMessage: {
+        minLength === 1 ?
+            "Please enter a value" :
+            `The value must be at least ${minLength} characters.`
```

Comment:
Can't rely on `errors` here because it doesn't exist. `errorMessage` defined inside validators have to be static.